compiler/record-minmax/CMakeLists.txt

```diff
@@ -23,30 +23,35 @@ if(NOT ENABLE_TEST)
   return()
 endif(NOT ENABLE_TEST)
 
+### 
+### record-minmax-for-thread-test is disabled, because gcc package has a bug.
+### (https://bugs.launchpad.net/ubuntu/+source/gcc-10/+bug/2029910)
+### Let's enable the target after the bug is fixed.
+###
 # Build record-minmax-for-thread-test if target arch is 64bit
 # Thread sanitizer is only available on 64bit machine
 # (https://github.com/google/sanitizers/wiki/ThreadSanitizerCppManual#supported-platforms)
-if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8")
-  # create record-minmax-for-thread-test target
-  # Note: record-minmax-for-thread-test is built with -fsanitize=thread so that thread sanitizer can check memory bugs,
-  # record-minmax is built without the option for performance.
-  add_executable(record-minmax-for-thread-test ${DRIVER} ${SOURCES})
-  target_include_directories(record-minmax-for-thread-test PRIVATE include)
-
-  target_link_libraries(record-minmax-for-thread-test arser)
-  target_link_libraries(record-minmax-for-thread-test safemain)
-  target_link_libraries(record-minmax-for-thread-test luci_import)
-  target_link_libraries(record-minmax-for-thread-test luci_env)
-  target_link_libraries(record-minmax-for-thread-test luci_export)
-  target_link_libraries(record-minmax-for-thread-test luci_interpreter)
-  target_link_libraries(record-minmax-for-thread-test dio_hdf5)
-  target_link_libraries(record-minmax-for-thread-test vconone)
-  target_link_libraries(record-minmax-for-thread-test nncc_coverage)
-  target_link_libraries(record-minmax-for-thread-test luci_log)
-
-  target_compile_options(record-minmax-for-thread-test PUBLIC -fsanitize=thread)
-  target_link_libraries(record-minmax-for-thread-test -fsanitize=thread)
-endif("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8")
+# if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8")
```

Comment:
Instead of adding `#` to this whole block,
can you revise this something like
```
if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8" AND False)
```

Reply:
We can use `CMAKE_CXX_COMPILER_VERSION` to disable under gcc-10.

```cmake
if("${CMAKE_SIZEOF_VOID_P}" STREQUAL "8" AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10)
  add_executable(record-minmax-for-thread-test ${DRIVER} ${SOURCES})
....

endif()

Reply:
@hseok-oh Thanks for the suggestion. I'd like to go with @seanshpark 's way, because we know `gcc-10` is ok, but not tested `gcc-11,12,13`. I think it would be safer to simply block the test for a while.